docs/content/projects/BuildTrace.md

```diff
@@ -9,25 +9,83 @@ This plugin collects tasks execution time in a trace event format.
 
 ## Getting started
 
-Apply the plugin in the root `build.gradle` file:
+Apply the plugin in the root buildscript:
 
-```groovy
+```kotlin
 plugins {
     id("com.avito.android.build-trace")
 }
+
+buildTrace {
+    enabled.set(true)
+}
 ```
 
 --8<--
 plugins-setup.md
 --8<--
 
-Run a build with `--profile` argument. You will get a message in a log:
+Run a build. You will get a message in a log:
 
 ```log
-Build trace: <path to the project>/outputs/trace/build.trace
+Build trace: <path to the project>/outputs/avito/build-trace/build.trace
 ```
 
-Open the file in `chrome://tracing`. 
+## Inspecting a trace
+
+The trace file can be opened by multiple tools.
+
+Chrome tracing
+
+`chrome://tracing`
+
+This is a legacy viewer.  
+
+Use WASD keys and search field for navigation.  
+
+[Perfetto](https://perfetto.dev/#viewer)
+
+This is a modern alternative for trace files.
+
+Here you can also make [analytical queries by SQL](https://perfetto.dev/docs/analysis/trace-processor):
+
+```sql
+-- Slowest tasks
+SELECT slice.name AS TASK_PATH, slice.dur / 1000000 AS DURATION_MS
+FROM slice
+ORDER BY slice.dur DESC
+```
+
+## Critical path
```

Comment:
An attached image would be helpful imo

Reply:
Update the first screenshot and added a reference to it.